packages/hpke/hpke.0.0.pre-alpha/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "HPKE implementation in OCaml"
+description: "RFC 9180 HPKE implementation in OCaml"
+maintainer: ["Tom Béné <himself@fantomebeig.net>"]
+authors: ["Tom Béné <himself@fantomebeign.net>"]
+license: "ISC"
+tags: ["cryptography" "hybrid encryption" "public key"]
+homepage: "https://github.com/FantomeBeignet/ohpke"
+doc: "https://www.fantomebeig.net/projects/ohpke/docs/"
+bug-reports: "https://github.com/FantomeBeignet/ohpke/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.7"}
+  "cstruct"
```

Comment:
You need a minimum version bound on Cstruct here (see the various failures in CI).